Documentation/project-docs/developer-guide.md

```diff
@@ -23,12 +23,11 @@ The tasks are represented in scripts (cmd/sh) in the root of the repo:
 * Clean
 * Sync
 * Build
-* Run Tests
-* Publish packages
+* Build-Tests
```

Comment:
These are logical steps but we do want them to directly map to script names I updated the casing to make that clearer.